addons/sale_timesheet/wizard/sale_make_invoice_advance.py

```diff
@@ -16,18 +16,19 @@ def _default_invoicing_timesheet_enabled(self):
         product_ids = order_lines.mapped('product_id').filtered(lambda p: p._is_delivered_timesheet())
         return bool(product_ids)
 
-    date_invoice_timesheet = fields.Date(
```

Comment:
We need a migration script to handle the data of this field removal.

Reply:
@tivisse , This field is in Transient Model. so it will not reinitialise new record each time? should we need to handle the data for Transient Model?

Thanks

Reply:
Mmmmh. You're right, the probability that the record is reused before being garbage collected is quite low. We can leave it like that then :)

Just make the runbot green then ;)

Reply:
@tivisse , Branch is green now. Thanks

Reply:
I've added the column initialization on the script too.